config/routes.rb

```diff
@@ -1,12 +1,6 @@
-# frozen_string_literal: true
-
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
-  # Reveal health status on /up that returns 200 if the app boots with no exceptions, otherwise 500.
-  # Can be used by load balancers and uptime monitors to verify that the app is live.
-  get 'up' => 'rails/health#show', as: :rails_health_check
-
-  # Defines the root path route ("/")
-  # root "posts#index"
+  resources :users, only: [:index, :show] do
+    resources :posts, only: [:index, :show]
+  end
 end
+
```

Comment:
**OPTIONAL** - Good job done so far configuring your controllers, but I suggest you have a root route for your application, this would serve as the landing page for your application. Please fix this 👍 👍